src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-items: center;
+
+  padding: 32px 16px 16px;
+  margin-bottom: 46px;
+
+  width: $card-width;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: #fff;
+
+  &:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
+
+  &__img {
+    margin-bottom: 40px;
+    max-height: 134px;
+
+    object-fit: contain;
+  }
+
+  &__title {
+    @extend %basic-text;
+
+    margin-bottom: 4px;
+
+    font-weight: 500;
+    color: $color-main-accent;
+  }
+
+  &__product-code {
+    @extend %small-text;
+
+    margin-bottom: 16px;
+
+    font-weight: 400;
+    color: $color-secondary;
+  }
+
+  &__price-box {
+    display: flex;
+    justify-content: space-between;
+
+    margin-bottom: 16px;
+  }
+
+  &__price-label {
+    @extend %basic-text;
+
+    font-weight: 400;
```

Comment:
It's the default value
![image](https://user-images.githubusercontent.com/91411692/222675732-2c874b59-93fa-4eae-9add-f57c0ce9ebcb.png)

![image](https://user-images.githubusercontent.com/91411692/222675648-684def54-5cf1-4c04-b707-21d0503a2dfb.png)

```suggestion
```